lntest/harness_assertion.go

```diff
@@ -667,6 +668,27 @@ func (h *HarnessTest) AssertStreamChannelForceClosed(hn *node.HarnessNode,
 	closingTxid := h.WaitForChannelCloseEvent(stream)
 	h.Miner.AssertTxInBlock(block, closingTxid)
 
+	// This makes sure that in case of a neutrino backend we do not have any
+	// lingering unconfirmed anchor cpfp transactions blocking some of our
+	// utxos.
+	if anchors && h.IsNeutrinoBackend() {
+		err := wait.NoError(func() error {
+			utxos := h.GetUTXOsUnconfirmed(hn,
+				lnwallet.DefaultAccountName)
+			total := len(utxos)
+			if total == 0 {
```

Comment:
Exactly, without removing the unmined transaction this check would fail especially when the wallet had enough utxos to create those independant anchor sweeps, as it is the case for this itest where each node is funded with 2 utxos each 1 btc in the beginning.